src/open_inwoner/accounts/tests/test_inbox_start_page.py

```diff
@@ -1,16 +1,25 @@
+import io
+
 from django.test import override_settings
 from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
 
 from django_webtest import WebTest
 from furl import furl
+from PIL import Image
 from privates.test import temp_private_root
+from webtest import Upload
+
+from open_inwoner.configurations.models import SiteConfiguration
 
 from ..models import Message
 from .factories import DocumentFactory, UserFactory
 
 
 @override_settings(ROOT_URLCONF="open_inwoner.cms.tests.urls")
 class InboxPageTests(WebTest):
+    csrf_checks = False
```

Comment:
This smells. 

Why wouldn't we want csrf_checks? Will the form work for users if we accidentally remove the csrf token from the template and the test still passes?